src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -145,23 +114,58 @@
 					@click="handleCreateConversation">
 					{{ t('spreed', 'Create conversation') }}
 				</NcButton>
-				<!-- Third page -->
-				<NcButton v-if="page===2 && (error || isPublic)"
-					ref="closeButton"
-					type="primary"
-					class="new-group-conversation__button"
-					@click="closeModal">
-					{{ t('spreed', 'Close') }}
-				</NcButton>
 			</div>
 		</NcModal>
+
+		<!-- Third page : this is the confirmation page-->
+		<NcModal v-else-if="page === 2"
+			:container="container"
+			@close="closeModal">
+			<NcEmptyContent>
+				<template #icon>
+					<Check v-if="!error && success && isPublic" :size="64" />
+					<div v-if="isLoading && !error && !success" class="icon-loading spinner" />
+					<AlertCircle v-if="error" :size="64" />
+				</template>
+
+				<template #description>
+					<p v-if="isLoading && !error && !success" class="confirmation__warning">
```

Comment:
Same v-if/v-else-if as with icons, to avoid accidental render of two icons/messages